README.md

```diff
@@ -36,22 +36,22 @@ Get Started
 -----------
 
 You'll want to get an account at [Parse](https://parse.com). After this you will
-be provided with three keys:
+be provided with two keys:
 
 * Application ID
-* JavaScript Key
```

Comment:
I did not see it used anywhere in the codebase so I removed it to provide simplicity.  If it's used in the future figured that would be when to add it.